app/services/proofing/lexis_nexis/ddp/response_redacter.rb

```diff
@@ -0,0 +1,152 @@
+          bb_bot_score
+          bb_fraud_rating
+          bb_fraud_score
+          champion_request_duration
+          digital_id
+          digital_id_confidence
+          digital_id_confidence_rating
+          digital_id_first_seen
+          digital_id_last_event
+          digital_id_last_update
+          digital_id_result
+          digital_id_trust_score
+          digital_id_trust_score_rating
+          digital_id_trust_score_reason_code
+          digital_id_trust_score_summary_reason_code
+          emailage.emailriskscore.billriskcountry
+          emailage.emailriskscore.correlationid
+          emailage.emailriskscore.domainexists
+          emailage.emailriskscore.domainrelevantinfo
+          emailage.emailriskscore.domainrelevantinfoid
+          emailage.emailriskscore.domainriskcountry
+          emailage.emailriskscore.domainrisklevel
+          emailage.emailriskscore.domainrisklevelid
+          emailage.emailriskscore.eaadvice
+          emailage.emailriskscore.eaadviceid
+          emailage.emailriskscore.eareason
+          emailage.emailriskscore.eareasonid
+          emailage.emailriskscore.eariskband
+          emailage.emailriskscore.eariskbandid
+          emailage.emailriskscore.eascore
+          emailage.emailriskscore.eastatusid
+          emailage.emailriskscore.emailexists
+          emailage.emailriskscore.first_seen_days
+          emailage.emailriskscore.firstverificationdate
+          emailage.emailriskscore.fraudrisk
+          emailage.emailriskscore.namematch
+          emailage.emailriskscore.phone_status
+          emailage.emailriskscore.responsestatus.errorcode
+          emailage.emailriskscore.responsestatus.status
+          emailage.emailriskscore.shipforward
+          emailage.emailriskscore.status
+          emailage.emailriskscore.totalhits
+          emailage.emailriskscore.uniquehits
+          enabled_services
+          event_datetime
+          event_type
+          fraudpoint.conversation_id
+          fraudpoint.friendly_fraud_index
+          fraudpoint.manipulated_identity_index
+          fraudpoint.product_status
+          fraudpoint.transaction_reason_code
+          fraudpoint.risk_indicators_codes
+          fraudpoint.risk_indicators_descriptions
+          fraudpoint.score
+          fraudpoint.stolen_identity_index
+          fraudpoint.suspicious_activity_index
+          fraudpoint.synthetic_identity_index
+          fraudpoint.transaction_status
+          fraudpoint.vulnerable_victim_index
+          integration_hub_results
+          org_id
+          policy
+          policy_details_api
+          policy_engine_version
+          policy_score
+          primary_industry
+          reason_code
+          request_duration
+          request_id
+          request_result
+          review_status
+          risk_rating
+          secondary_industry
+          service_type
+          session_id
+          session_id_query_count
+          ssn_hash
```

Comment:
This is my fault for not catching it in the doc: ssn_hash should not be on the allow list

Reply:
cool... thanks!